dashboard/src/reducers/config.ts

```diff
@@ -0,0 +1,30 @@
+import { getType } from "typesafe-actions";
+
+import actions from "../actions";
+import { ConfigAction } from "../actions/config";
+import { IConfig } from "../shared/Config";
+
+export interface IConfigState extends IConfig {
+  loaded: boolean;
+}
+
+const initialState: IConfigState = {
+  loaded: false,
+  namespace: "",
+};
+
+const configReducer = (state: IConfigState = initialState, action: ConfigAction): IConfigState => {
+  switch (action.type) {
+    case getType(actions.config.requestConfig):
```

Comment:
Not for this patch but in general we should start thinking of error handling so we do not run into cases like this one #433